packages/mgt-components/src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -635,57 +635,57 @@ export class MgtPersonCard extends MgtTemplatedComponent {
     let email: TemplateResult;
     if (getEmailFromGraphEntity(person)) {
       email = html`
-         <div class="icon"
-           @click=${() => this.emailUser()}
-           @mouseenter=${() => this.setHoveredState('email', true)}
-           @mouseleave=${() => this.setHoveredState('email', false)}
-           tabindex=0
-           role="button">
-           ${this.isHovered('email') ? getSvg(SvgIcon.SmallEmailHovered) : getSvg(SvgIcon.SmallEmail)}
-         </div>
-       `;
+        <fluent-button class="icon"
```

Comment:
Because we're using an SVG as the button content we need to ensure that there is a accessible description for the button. This button needs an aria-label which should be something like:
```
`${strings.emailButtonLabel} ${user.displayName}`
```

This comment applies to all of these buttons